### ISSUE TITLE
no results for infohash lookup

### ISSUE THREAD

**Issue by danoctavian** [feross/bittorrent-dht#57]
The following code snippet yields no peers for the lookup query. no peer is printed to stdout. IF i use the same DHT object for announce and lookup, the correct IP is found. However when creating a new DHT object, the IP is not found. 

Am i misusing the library or is this expected behavior and why?

Thanks!
```javascript
var DHT = require("bittorrent-dht")

var ih = "46f9e9a430f839e7acd76733d649b885985cff4b"
var dht =  new DHT()

dht.listen(20000, function () {
  console.log('now listening')
})


dht.on('ready', function () {
  console.log("dht is ready")

  dht.announce(ih, 6666, function (r) {
    var otherDHT = new DHT()

    console.log("announce result is " + r)

    otherDHT.listen(20001, function () {
    })
    otherDHT.on('ready', function () {
      console.log("other dht is ready")

      otherDHT.lookup(ih, function (err, r) {
      })
      otherDHT.on('peer', function (addr, ih, from) {
        console.log("got peer " + addr)
      })
    })
  })
})
```

**Comment by feross** [comment]
Hm, this should work. Not sure why it doesn't.

**Comment by feross** [comment]
This issue was caused by a bug introduced in @mvayngrib's PR #52, and fixed in #54.

Will release a new version of `bittorrent-dht` to fix the issue.

**Comment by feross** [comment]
Released as 3.0.0.

**Comment by danoctavian** [comment]
nice work, thanks!
